chemist/src/main/scala/flask.scala

```diff
@@ -76,21 +99,16 @@ object Flask {
    * Call out to the specific location and grab the list of things the flask
    * is already mirroring.
    */
-  private def requestAssignedTargets(location: Location)(http: dispatch.Http): Task[Set[Target]] = {
+  private def requestAssignedTargets(location: Location)(http: dispatch.HttpExecutor): Task[Set[Target]] = {
     import argonaut._, Argonaut._, JSON._, HJSON._
     import dispatch._, Defaults._
 
     val a = location.uriFromTemplate(LocationTemplate(s"http://@host:@port/mirror/sources"))
     val req = Task.delay(url(a.toString)) <* Task.delay(log.debug(s"requesting assigned targets from $a"))
-    req flatMap { b =>
-      http(b OK as.String).map { c =>
-        Parse.decodeOption[List[Cluster]](c
-        ).toList.flatMap(identity
-        ).map { cluster =>
-          log.debug(s"Received cluster $cluster from $a")
-          cluster
-        }.foldLeft(Set.empty[Target]){ (a,b) =>
-          a ++ b.urls.map(s => Target(b.label, new URI(s))).toSet
+    req flatMap {
```

Comment:
what do you want to change? Restore linebreak after the "=>"?
Or break to have one function call per line?

I do not see consistent code style, similar code style was in `gatherAssignedTargets` before i changed and it is easier to follow for me (i can see input and function on the same line). I can revert it if you fill strong about it.

The code change here is use of `flatten` instead of `flatMap{identity}` and removal of block with log message.